tce/tools/scripts/install_llvm_9.sh

```diff
@@ -94,14 +78,13 @@ cd build
 # https://reviews.llvm.org/D54978#1390652
 # You might also need to delete libz3-dev.
 # This appears at least with Ubuntu 18.04.
-
-cmake -G "Unix Makefiles" \
+cmake ../llvm/ -DLLVM_ENABLE_PROJECTS=clang \
+    -G "Unix Makefiles" \
     $LLVM_BUILD_MODE\
     -DCMAKE_INSTALL_PREFIX=$TARGET_DIR \
     -DLLVM_LINK_LLVM_DYLIB=TRUE \
     -DLLVM_ENABLE_RTTI=TRUE \
     -DLLVM_ENABLE_Z3_SOLVER=OFF \
-    .. \
     || eexit "Configuring LLVM/Clang failed."
 make -j8 CXXFLAGS="-std=c++11" REQUIRES_RTTI=1 \
```

Comment:
Can you test if -std=c++11 is still needed?

Reply:
LLVM definitely uses C++11 features. 

Whether -std=c++11 is needed or not depends on the defaults of the compiler which is used for compiling the llvm(g++ that comes with the distribution). Testing and possibly removing it from here (if works at some system without it) just causes a possible "works on my machine"-problem were it starts to fail when compiled with older/different linux distritbutions which have older version of the default g++ compiler.

Reply:
OK. I know it does use the features therefore I guessed it'd add std=c++11 by default nowadays. But shouldn't harm indeed.